langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -786,15 +812,15 @@ function testFromJsonWithTypeMap() {
         title: "Some",
         year: 2010
     };
-    map<anydata>|error movieMap = movie.fromJsonWithType(MapOfAnyData);
+    map<anydata> movieMap = checkpanic movie.fromJsonWithType(MapOfAnyData);
     map<anydata> movieMap2 = <map<anydata>> movieMap;
```

Comment:
```suggestion
    map<anydata> movieMap2 = checkpanic movie.fromJsonWithType(MapOfAnyData);
```

Reply:
Please check other changes in this file too.

Reply:
Fixed